index.html

```diff
@@ -64,33 +63,60 @@ <h2>
     <p>
       TBW
     </p>
+    <p>
+      TBW: <dfn>Example credential types</dfn>
+    </p>
     <h2>
       Model
     </h2>
-    <dl class="definitions" data-sort="" data-cite="vc-data-model">
+
+    <p class="note" title="Definitions under discussion">
+The goal of the definitions in this section is to reuse or establish
+terminology that is common across a variety of digital credential formats and
+protocols. Discussions surrounding these definitions are active and
+the definitions are likely to change over the next several months.
+    </p>
+
+    <dl class="definitions" data-sort="" data-cite="vc-data-model-2.0">
       <dt>
         <dfn>Digital credential</dfn>
       </dt>
       <dd>
-        <p>
-          Is a [=verifiable credential=] about a person.
-        </p>
+        A cryptographically signed digital document containing one or more
+        [=claims=] made by an [=issuer=] about one or more [=subjects=].
+      </dd>
+      <dt>
+        <dfn>Credential manager</dfn>
+      </dt>
+      <dd>
+        Software that provides a user interface for querying, selecting, and
+        establishing consent for the [=presentation=] of a [=digital
+        credential=]. Examples include a digital wallet, or a login manager,
+        that manages various [=example credential types|credential types=].
+      </dd>
+      <dt>
+        <dfn data-dfn-for="digital credential">query</dfn>
+      </dt>
+      <dd>
+        A format that a [=verifier=] uses, via an [=digital credential/exchange
```

Comment:
```suggestion
        A format that a [=verifier=] or [=user agent=] uses, via an [=digital credential/exchange
```

So, one of the big architectural elements here is that, at least currently in Chrome/Android, the "user agent" (the browser or the operating system), does the credential selection before handing the request to the holder. So, the "query" is actually *also* used by the [=user agent=] to filter out which holders would be able to answer the query.

Reply:
Good point!

Reply:
Yes, right, I'm going to merge your change suggestion in @samuelgoto, but wanted to make a distinction before doing so.

In the many years that led up to W3C Verifiable Credentials, we found that it was useful to separate the "roles" in the ecosystem (e.g. "user agent") from the software that performs actions on behalf of those roles (e.g., "user agent").

The roles that seem to be common to all these digital credential ecosystems are: issuer, holder, verifier.

Each of those roles uses software to achieve their goals in the ecosystem. Issuers (role) use issuance software (software agent) to issue verifiable credentials. Holders (role) use digital wallets (software agent) to manage verifiable credentials. Verifiers (role) use verification software (software agent) to verify verifiable credentials. A single entity can play many roles in a particular workflow -- for example, an organization might request prerequisite VCs as a verifier before it issues a subsequent VC that is dependent on the prerequisite VCs.

That approach has served us well over the past decade, and it's not a new concept, most other software ecosystems are explained in this manner.

All this to say, we shouldn't conflate the "verifier" (role) with the "user agent" (software agent). So, if we have agreement on the above, your change suggestion would switch to the following, @samuelgoto:

```suggestion
        A format that [=verifier=] software or a [=user agent=] uses, via an [=digital credential/exchange
```

I know the change feels pedantic, but we've always gotten ourselves into trouble in the other groups when we weren't this precise with our language.

Thoughts on the above before I commit the update?

Reply:
FWIW, @msporny's (...dare I say it...) "ontology" 🙈 of people's roles and software agents is also the one that I've been subscribing to.

Reply:
>  "ontology" 🙈

booo! Throw that heretic into the streets! There's no room for that sort of language in the digital credentials spec! :stuck_out_tongue_winking_eye:

Reply:
I'm sorry @samuelgoto, I wasn't trying to disparage your good name! I was only trying to disparage @marcoscaceres! :P

@samuelgoto's snippet applied in e162c9e2502266508891e99b58603c100980e4df.